credsweeper/app.py

```diff
@@ -227,6 +243,63 @@ def file_scan(self, content_provider: ContentProvider) -> List[Candidate]:
 
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
+    def data_scan(self, data_provider: DataContentProvider, depth: int, recursive_limit: int) -> List[Candidate]:
+        """Recursive function to scan files which might be containers like ZIP archives
+
+            Args:
+                data_provider: DataContentProvider object may be a container
+                depth: maximal level of recursion
+                recursive_limit: maximal bytes of opened files to prevent recursive zip-bomb attack
+        """
+        candidates: List[Candidate] = []
+        logging.debug(f"Start scan data: {data_provider.file_path} {len(data_provider.data)} bytes")
+
+        if 0 > depth:
+            # break recursion if maximal depth is reached
+            logging.error(f"bottom reached {data_provider.file_path} recursive_limit:{recursive_limit}")
```

Comment:
Is it good to logging as error..? I think there are many cases that over `recursive_limit`..
So how about set logging level as a debug?
```suggestion
            logging.debug(f"bottom reached {data_provider.file_path} recursive_limit:{recursive_limit}")
```